### ISSUE TITLE
Cannot find module 'inquirer'

### ISSUE THREAD

**Issue by AhmedGamal-Inmobly** [Samsung/cordova-plugin-toast#55]
I'm getting Cannot find module 'inquirer' with grunt commands when trying to prepare or build. 
What am I missing ?

**Comment by prozanne** [comment]
Hi :D
In this case if inquirer module is not installed, It might be shown error such as "Cannot find module 'inquirer'".
Can you check again whether you installed all dependent modules with 'npm install' command.
Can you check whether "inquire" directory is exist on your [project root]/node_modules.
![image](https://user-images.githubusercontent.com/1733182/28144551-1103bfd4-67a7-11e7-93d3-aff841e5a289.png)

If all above things are good, please upload your application as zip file.

**Comment by AhmedGamal-Inmobly** [comment]
Yes I've check the dependencies and inquirer is there, here's the app 
https://drive.google.com/file/d/0B8lbcIOZH70ERDgwN0doQ21nbTA/view?usp=sharing

**Comment by dholbrook86** [comment]
Did you get this figured out? I'm having the same issue. "inquirer" is in my node_modules folder. I've deleted and reinstalled the node_modules folder to no avail. What should I do?

**Comment by pwsses** [comment]
Hi 
Please check "create empty project" in Wiki
url : https://github.com/Samsung/cordova-plugin-toast/wiki/Create-Project
If you did it, you can find "inquirer" in node_modules folder.

# Create cordova project
$ cordova create TestApp
$ cd TestApp

# Beware of hidden file
$ cp -rf ../grunt-cordova-sectv/sample/. ./
$ npm install ../grunt-cordova-sectv
# Grunt task for build and package

# Install dependency modules
$ npm install

# For toast browser simulator
$ cordova platform add browser

# Mandatory plugins for using browser simulator (not for other platforms)
$ cordova plugin add cordova-plugin-device
$ cordova plugin add cordova-plugin-network-information
$ cordova plugin add cordova-plugin-globalization

# Add toast plugin
$ cordova plugin add ../cordova-plugin-toast

**Comment by AhmedGamal-Inmobly** [comment]
@dholbrook86 I had to execute `npm install inquirer` in all the repos in toast folder and it worked

**Comment by dholbrook86** [comment]
@AhmedGamal-Inmobly I did that. So I thought. I really thought I tried everything. Ended up moving on to something else. I was just experimenting anyway. I will have to come back to it later. Thanks for responding though! I appreciate it.

**Comment by enjikaka** [comment]
@pwsses Following those guidelines; I still end up with `Warning: Cannot find module 'inquirer' Use --force to continue.` when running `grunt sectv-prepare:tv-webos` and related commands. The module is in node_modules directory.

**Comment by rikoopa** [comment]
i have the same issue
i installed in all repos the following
- inquirer
- zip-dir
- xml2js
- js2xmlparser
- mustache

**Comment by cnavarroestrella** [comment]
@rikoopa Thanks for this. Actually, I just needed to do this in `toast/grunt-cordova-sectv`, so, for anyone coming later:

```bash
cd grunt-cordova-sectv
npm install inquirer xml2js mustache js2xmlparser zip-dir
```

This seems to get rid of the issue.

**Comment by Titella** [comment]
After solving "inquirer", now I get an error **Unexpected token {**...
Does the same thing happen to you?
Any ideas ?
I think it has to do with the grunt-cordova-sectv that does not work


### Initializingte Bexfy\SmartTV\template TOAST\TemplateToast\TestApp>
Command-line options: --verbose, --force

Reading "Gruntfile.js" Gruntfile...OK

Registering Gruntfile tasks.
Initializing config...OK

Registering "grunt-contrib-clean" local Npm module tasks.
Reading C:\Projecte Bexfy\SmartTV\template TOAST\TemplateToast\TestApp\node_modules\grunt-contrib-clean\package.json...OK
Parsing C:\Projecte Bexfy\SmartTV\template TOAST\TemplateToast\TestApp\node_modules\grunt-contrib-clean\package.json...OK
Loading "clean.js" tasks...OK
+ clean

Registering "grunt-cordova-sectv" local Npm module tasks.
Reading C:\Projecte Bexfy\SmartTV\template TOAST\TemplateToast\TestApp\node_modules\grunt-cordova-sectv\package.json...OK
Parsing C:\Projecte Bexfy\SmartTV\template TOAST\TemplateToast\TestApp\node_modules\grunt-cordova-sectv\package.json...OK
Loading "sectv-build.js" tasks...OK
+ sectv-build
Loading "sectv-prepare.js" tasks...OK
+ sectv-prepare
Loading "Gruntfile.js" tasks...OK
>> No tasks were registered or unregistered.

Running tasks: sectv-build

Running "sectv-build" task

Running "sectv-build:sectv-orsay" (sectv-build) task
Verifying property sectv-build.sectv-orsay exists in config...OK
Files: -> platforms/sectv-orsay/build
Warning: Unexpected token { Used --force, continuing.
